.gitignore

```diff
@@ -15,3 +15,4 @@ subgrounds.egg-info/
 
 # apple
 .DS_Store
+.venv*/
```

Comment:
We shouldn't need to do this, there should already be a `.gitignore` in your `.venv`.

Reply:
there was not so I had to add it in. 

Do you also mean that the `.gitignore` I have should not be getting pushed?

Reply:
The `.gitignore` is auto-generating if the `.venv` is created via python's built in `venv` module (which is also what `poetry` uses for generating environments).